### PR TITLE
2025-nhi-long-lived-secrets

### PR DESCRIPTION
**Pull request by bar-orca**
Small update for risk #5 and adding risk #7

### CODE REVIEW

File: 2025/docs/7-long-lived-secrets.md [hunk 1]
```diff
@@ -1 +1,45 @@
-TBA
\ No newline at end of file
+# NHI5:2024 Overprivileged NHI
```
Comment (ronilichtman) [inline 2025/docs/7-long-lived-secrets.md:1]: Please update the title

**Review by ronilichtman — COMMENTED** [review]
Looks great! One small comment

File: 2025/docs/7-long-lived-secrets.md (hunk 1 above)
Comment (bar-orca) [reply 2025/docs/7-long-lived-secrets.md]: fixed both

**Review by ronilichtman — APPROVED** [review]
LGTM! 🔥 🙌

**Pull request merged**